packages/clang-format-git-python/package.json

```
@@ -45,12 +45,12 @@
     "node": ">=16"
   },
   "scripts": {
-    "postinstall": "test -d src && npm run build || npm run chmod",
+    "postinstall": "shx test -d src && npm run build || npm run chmod",
     "prepublishOnly": "npm run build",
-    "build": "npx babel src -d build && npx tsc && cp -r src/script build && cp ../../LICENSE.md ../../README.md .",
+    "build": "npx babel src -d build && npx tsc && shx cp -r src/script build && shx cp ../../LICENSE.md ../../README.md .",
     "postbuild": "npm run chmod",
     "test": "node --test",
-    "chmod": "find ./src/script ./build/script -type f -exec chmod 755 {} + || exit 0",
+    "chmod": "shx chmod 755 ./src/script/**/* ./build/script/**/* || exit 0",
     "dev": "node src/cli.js",
     "start": "node build/cli.js"
   },
```

```suggestion
  },
  "dependencies": {
    "shx": "^0.3.4"
  }
```

`shx` should be in the `dependencies` as it used in production. Please run  `npm install shx -w packages/clang-format-git-python` to add `shx` as `dependencies`.